src/modals/MediaDbFolderImportModal.ts

```diff
@@ -74,6 +76,20 @@ export class MediaDbFolderImportModal extends Modal {
 		});
 		contentEl.appendChild(titleFieldNameComponent.inputEl);
 
+		contentEl.createDiv({ cls: 'media-db-plugin-spacer' });
+		contentEl.createEl('h3', { text: "Name of 'id' metadata field to use in API query (if present, will be used instead of title)." });
+
+		const idFieldNameComponent = new TextComponent(contentEl);
+		idFieldNameComponent.inputEl.style.width = '100%';
```

Comment:
I would normally say: "Styles should be added with CSS classes." But the code above also doesn't use CSS classes, so this is fine.